### PR TITLE
[KYUUBI #340] Swtich to commons-lang3

### LINKED ISSUE #340: Avoiding using commons-lang2
**Describe the bug**
There are some usage of `commons-lang2` in `kyuubi-common`, we need to migrate them to `commons-lang3`

**To Reproduce**
1. Configurations
2. Environments
3. Operations

**Expected behavior**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #341](https://github.com/yaooqinn/kyuubi/pull/341)    [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

close #340 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-common/pom.xml [hunk 1]
```diff
@@ -59,6 +59,11 @@
             <artifactId>hadoop-common</artifactId>
         </dependency>
 
+        <dependency>
```
Comment (yaooqinn) [inline kyuubi-common/pom.xml:62]: do we also need to exclude common2/3 from  other deps?

Comment (pan3793) [reply kyuubi-common/pom.xml:62]: I'm not sure `hadoop-common` works without `common2`

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.1.0
